README.md

```diff
@@ -1 +1,107 @@
+
+  File: saml20-idp-hosted.php
+  
+      $metadata['globus-idp'] = array(
+        /*
+        * The hostname for this IdP. This makes it possible to run multiple
+        * IdPs from the same configuration. '__DEFAULT__' means that this one
+        * should be used by default.
+        */
+        'host' => '__DEFAULT__',
+
+        /*
+        * The private key and certificate to use when signing responses.
+        * These are stored in the cert-directory.
+        */
+        'privatekey' => '[[YOUR_PRIVATE_KEY]].pem',
+        'certificate' => '[[YOUR_PRIVATE_CERT]].crt',
+
+        /*
+        * The authentication source which should be used to authenticate the
+        * user. This must match one of the entries in config/authsources.php.
+        */
+        'auth' => 'globus'
+      );
+
+  File: saml20-idp-remote.php
+  
+        $metadata['globus-idp'] = array (
+          'metadata-set' => 'saml20-idp-remote',
+          'entityid' => 'globus-idp',
+          'SingleSignOnService' =>
+              array (
+                  0 =>
+                  array (
+                      'Binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                      'Location' => 'https://your.endpoint/simplesaml/saml2/idp/SSOService.php',
+                  ),
+              ),
+          'SingleLogoutService' =>
+              array (
+                  0 =>
+                  array (
+                      'Binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                      'Location' => 'https://your.endpoint/simplesaml/saml2/idp/SingleLogoutService.php',
+                  ),
+              ),
+          'keys' =>
+        array (
+          0 =>
+          array (
+            'encryption' => true,
+            'signing' => true,
+            'type' => 'CERTTYPE',
+            'CERTTYPE' => '...',
+          ),
+          1 =>
+          array (
+            'encryption' => true,
+            'signing' => true,
+            'type' => 'CERTTYPE',
+            'CERTTYPE' => '...',
+          ),
+        ),
+          'NameIDFormat' => 'urn:oasis:names:tc:SAML:2.0:nameid-format:transient',
+        );
+        
+## Suggestions, Bugfixes, Feedback
+
+  While this module is used in the XDMoD Project, it is not officially supported as part of the overarching project or maintained as such. Tickets submitted to XDMoD support pertaining to this module cannot be supported, and are likely to go unanswered. Pull requests are welcome, but turnaround might be intermittent in lieu of other team priorities. 
+  
+  For feedback, commentary, requests, please reach out to the author/maintainer, [Rudra Chakraborty](mailto:rudracha@buffalo.edu).
```

Comment:
Probably best to use the ccr-help email address here so that anyone can help with the support requests.

Reply:
We're not supporting the module as a part of XDMod or CCR, it's more a one-off. Using the ccr-help email implies a guarantee of support.